fe/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -1438,6 +1476,22 @@ public boolean returnsSingleRow() {
         return false;
     }
 
+    @Override
+    public boolean containsCorrelatedPredicate() {
+        // check inline view
+        for (TableRef tableRef : fromClause_.getTableRefs()) {
+            if (tableRef instanceof InlineViewRef) {
+                if (((InlineViewRef) tableRef).getViewStmt().containsCorrelatedPredicate()) {
+                    return true;
+                }
+            }
+        }
+        if (whereClause == null) {
+            return false;
+        }
+        return whereClause.isCorrelatedPredicate(getTableRefIds());
```

Comment:
why do you only analyze where clause？ what other clasue？for example having clasue，selectList。

Reply:
This function has been removed.